src/algebra/module/pi.lean

```diff
@@ -60,20 +60,20 @@ instance is_scalar_tower'' {g : I → Type*} {h : I → Type*}
 
 @[to_additive]
 instance smul_comm_class {α β : Type*}
-  [Π i, has_scalar α $ f i] [Π i, has_scalar β $ f i] [∀ i, smul_comm_class α β (f i)] :
+  [Π i, has_scalar α $ f i] [Π i, has_scalar β $ f i] [Π i, smul_comm_class α β (f i)] :
```

Comment:
This change is unhelpful, `∀` is more correct than `Π` because `smul_comm_class _ _ _ : Prop`. The rule is "use what lean would use to pretty-print", which in turn is "use `∀` for Sort 0, `Π` for everything else"